packages/contracts/test/contracts/fractal_registry/FractalRegistry.ts

```diff
@@ -47,6 +47,13 @@ describe("FractalRegistry", () => {
 
       expect(await subject.getFractalId(user1.address)).to.equal(ID_42);
     });
+
+    it("is gone after remove", async () => {
+      await subject.addUserAddress(user1.address, ID_42);
+      await subject.removeUserAddress(user1.address);
+
+      expect(await subject.getFractalId(user1.address)).to.equal(bytes32(""));
+    });
```

Comment:
can you add a spec to ensure a non-allowed account can't actually remove anything?

Reply:
oh sorry, I somehow missed that one